Input.cpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+#include "Engine.h"
+using namespace sf;
+
+
+void Engine::input()
+{
+	Event event;

```

Comment:
The `event` is not used, can it be removed?

Reply:
Yup, it can be removed. Not sure what I wanted to do with it here, so I'll keep it in mind as I finish the HUD and TimerBar..